Source/Scene/Cesium3DTileset.js

```diff
@@ -1223,7 +1239,10 @@ define([
 
         var removeFunction = removeFromProcessingQueue(tileset, tile);
         tile.contentReadyToProcessPromise.then(addToProcessingQueue(tileset, tile));
-        tile.contentReadyPromise.then(removeFunction).otherwise(removeFunction);
+        tile.contentReadyPromise.then(function() {
+            removeFunction();
+            tileset.tileLoad.raiseEvent(tile);
+        }).otherwise(removeFunction);
```

Comment:
We would probably add a `tileFailed` event instead, a little more detail here: https://github.com/AnalyticalGraphicsInc/cesium/issues/5158